workers/data_refinery_workers/downloaders/management/commands/queue_downloader.py

```diff
@@ -33,18 +34,27 @@ def handle(self, *args, **options):
             survey_job=survey_job,
             source_type="ARRAY_EXPRESS",
             size_in_bytes=0,
-            download_url="http://www.ebi.ac.uk/arrayexpress/files/E-MTAB-3050/E-MTAB-3050.raw.1.zip",  # noqa
-            raw_format="MICRO_ARRAY",
+            download_url="ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/GEOD/E-GEOD-59071/E-GEOD-59071.raw.3.zip",  # noqa
+            raw_format="CEL",
             processed_format="PCL",
-            pipeline_required="MICRO_ARRAY_TO_PCL",
-            accession_code="A-AFFY-1",
-            internal_location="A-AFFY-1/MICRO_ARRAY_TO_PCL/",
-            organism=1,
+            pipeline_required="AFFY_TO_PCL",
+            platform_accession_code="A-AFFY-141",
+            experiment_accession_code="E-GEOD-59071",
+            experiment_title="It doesn't really matter.",
+            name="GSM1426072_CD_colon_active_2.CEL",
+            internal_location="A-AFFY-141/AFFY_TO_PCL/",
+            organism_id=9606,
+            organism_name="HOMO SAPIENS",
+            release_date="2017-05-05",
+            last_uploaded_date="2017-05-05",
             status=BatchStatuses.NEW.value
         )
         batch.save()
 
-        downloader_job = DownloaderJob(batch=batch)
+        downloader_job = DownloaderJob()
         downloader_job.save()
+        downloader_job_to_batch = DownloaderJobsToBatches(batch=batch,
+                                                          downloader_job=downloader_job)
+        downloader_job_to_batch.save()
         logger.info("Queuing a task.")
         download_array_express.delay(downloader_job.id)
```

Comment:
That's exactly what's going on here. `@shared_task` turns the function into an instance of Celery's Task class.